corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -0,0 +1,146 @@
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.es.case_search import CaseSearchES
+import timeit
+import json
+
+
+# Returns min, max, average of test suite results list
+def get_stats(results_list):
+    n = len(results_list)
+    if n == 1:
+        return results_list[0], results_list[0], results_list[0]
+    min_value = min(results_list)
+    max_value = max(results_list)
+    avg_value = sum(results_list) / n
+    return min_value, max_value, avg_value
+
+
+def run_baseline_query(query_obj):
+    print("Running baseline query...")
+    result = query_obj.run().total
```

Comment:
since you only care about the total you can do a count query here:
```suggestion
    result = query_obj.count()
```